tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
Instead of changing the top-level tsconfig, lets create another one in the scripts folder that extends this one.

Reply:
Agreed

Reply:
I have create the application.json and scripts.json, found out that the property could not be in the base file, so I needed to create an application config as well. Let me know if it is good for you.